security/cert_manager_operator/cert-manager-authenticate-gcp.adoc

```diff
@@ -0,0 +1,16 @@
+:_content-type: ASSEMBLY
+[id="cert-manager-authenticate-gcp"]
+= Authenticating the {cert-manager-operator} with GCP Workload Identity
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-authenticate-gcp
+
+toc::[]
+
+You can authenticate the {cert-manager-operator} on the GCP Workload Identity cluster by using the cloud credentials. You can configure the cloud credentials by using the `ccoctl` binary.
+
+include::modules/cert-manager-configure-cloud-credentials-gcp-sts.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
```

Comment:
The additional resources heading needs an anchor ID in addition to the role statement. For example:

```
[role="_additional-resources"]
[id="additional-resources_cert-manager-authenticate-gcp"]
== Additional resources
```

source: [OCP docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies)